### ISSUE TITLE
MAUI iOS charts prevent scrolling and tooltip visibility

### ISSUE THREAD

**Issue by jai-io** [beto-rodriguez/LiveCharts2#1289]
**Bug Description**
When a line-based chart is situated within a scrollview, attempting to initiate a scroll directly from the chart seems to disable any scrolling functionality. Additionally, interactions with individual points within the chart, such as clicks, are not recognized, thus failing to activate the tooltip. However, dragging actions within the chart area do successfully activate the tooltip.

**To Reproduce**
XAML:
```
<ContentPage.Content>
        <ScrollView>
            <Grid>
                <Grid.RowDefinitions>
                    <RowDefinition Height="300" />
                    <RowDefinition Height="300" />
                    <RowDefinition Height="300" />
                </Grid.RowDefinitions>
                <Grid Grid.Row="0">
                    <maui:CartesianChart
                        ZoomMode="None"
                        Series="{Binding Series}"
                        Title="{Binding Title}" />
                </Grid>
                <Grid Grid.Row="1" Padding="0,100">
                    <Label Text="Section 1 - Test Scrolling"></Label>
                </Grid>
                <Grid Grid.Row="2" Padding="0,100">
                    <Label Text="Section 2 - Test Scrolling"></Label>
                </Grid>
            </Grid>
        </ScrollView>
</ContentPage.Content>
```
    
Viewmodel:
```
public ISeries[] Series { get; set; } =
    {
        new LineSeries<double>
        {
            Values = new double[] { 2, 1, 3, 5, 3, 4, 6 },
            Fill = null
        }
    };

public LabelVisual Title { get; set; } =
        new LabelVisual
        {
            Text = "My chart title",
            TextSize = 25,
            Padding = new LiveChartsCore.Drawing.Padding(15),
            Paint = new SolidColorPaint(SKColors.DarkSlateGray)
        };
```
   
**Expected Behaviour**
Dragging on the chart should allow scrolling functionality. Additionally, clicking on individual data points within the chart should activate the tooltip.

**Device and Versions**
iOS Emulator Version: 15.0 (1015.2)
iOS Operating System: 17.0
MAUI:  8.0.0-rc.1.9171/8.0.100-rc.1
.NET: .NET 8.0.100-rc.1.23463.5

**Comment by hwsmo** [comment]
Hi, I can also confirm this bug.

**Comment by beto-rodriguez** [comment]
Have you tried this on Android? it seems to work properly on Android for me.

**Comment by hwsmo** [comment]
You're right, there's no problem on Android. The bug seems to only exist on iOS.

**Comment by beto-rodriguez** [comment]
This is not working properly in the lates release.

This issue is not completely fixed on Android, but we can track the Android fix at #1343
